lib/bolt/project_migrator.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'bolt/project_migrator/config'
+require 'bolt/project_migrator/inventory'
+require 'bolt/project_migrator/modules'
+
+module Bolt
+  class ProjectMigrator
+    def initialize(config, outputter)
+      @config    = config
+      @outputter = outputter
+    end
+
+    def migrate
+      ok = migrate_inventory && migrate_config
+
+      if ok && ENV['BOLT_MODULE_FEATURE']
+        ok = migrate_modules
```

Comment:
This should be a new `modules_ok` var so we don't overwrite if any of the previous migrations failed. Then on line 21, should be `if ok && modules_ok`. Right now this will print that the project migrated successfully even if migrate_inventory or migrate_config returns false.

Reply:
I don't think that's accurate. if either `migrate_inventory` or `migrate_config` is `false`, then `migrate_modules` will never be called since the condition `ok && ENV['BOLT_MODULE_FEATURE']` is never met.

Reply:
Oh yes, I missed the 'ok &&' conditional.